### PR TITLE
Bubblechat Nil Checks

### PR DESCRIPTION
**Pull request by Stickmasterluke**

### CODE REVIEW

File: CoreScriptsRoot/CoreScripts/BubbleChat.lua [hunk 1]
```diff
@@ -334,7 +334,7 @@ local function createChatOutput()
 	end
 
 	function this:CreateBillboardGuiHelper(instance, onlyCharacter)
-		if not this.CharacterSortedMsg:Get(instance)["BillboardGui"] then
+		if instance and not this.CharacterSortedMsg:Get(instance)["BillboardGui"] then
```
Comment (hockeychaos) [inline CoreScriptsRoot/CoreScripts/BubbleChat.lua]: what happens if we just return if instance is nil?

File: CoreScriptsRoot/CoreScripts/BubbleChat.lua [hunk 2]
```diff
@@ -133,7 +133,7 @@ local function createMap()
 	end
 
 	function this:Get(key)
-		if not this.data[key] then
+		if key and not this.data[key] then
```
Comment (hockeychaos) [inline CoreScriptsRoot/CoreScripts/BubbleChat.lua]: if key is nil, can we just return nil?

File: CoreScriptsRoot/CoreScripts/BubbleChat.lua (hunk 1 above)
Comment (hockeychaos) [inline CoreScriptsRoot/CoreScripts/BubbleChat.lua]: bump

Comment (Stickmasterluke) [inline CoreScriptsRoot/CoreScripts/BubbleChat.lua]: No billboard is made or updated

**Pull request merged**